test/tests/pbs_smoketest.py

```diff
@@ -1251,6 +1251,8 @@ def delete_resource_helper(self, r, t, f, c, obj_type, obj_id):
                 if obj_type == JOB:
                     j = Job(TEST_USER1, {'Resource_List.' + r: val})
                 else:
+                    print "======= Resource being set on the reservation:" +\
+                        str(r) + ":" + str(val)
```

Comment:
Don't use print, use self.logger.info() or self.logger.debug()

Reply:
oh my bad, this got left in, I was using it for debugging